hw3.1/style.css

```diff
@@ -0,0 +1,125 @@
+.banner{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+header{
+    background-color: rgb(154, 9, 9);
+    height: 50px;
+}
+.images{
+    background-color: black;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.big{
+    width: 220px;
+    height: 160px;
+}
+div.images{
+    height: 200px;
+}
+ul.top1{
+    text-align: center;
+    list-style-type: none;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    font-size: 18px;
+}
+.link{
+    text-decoration: none;
+    color: white;
+}
+li{
+    display: inline;
+}
+nav.top{
+    background-color: black;
+    margin-top: -25px;
+}
+/*div.navigation{
+    display: flex;
+    justify-items: flex-start;
+    width: 200px;
+}
+ul.left{
+display: flex;
+flex-direction: column;
+}
+*/
+div.parents{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+    margin-top: 195px;
+    background-color: rgb(154, 9, 9);
+    height: 450px;
+}
+div.container1{
+    background-color: rgb(230, 228, 228);
+    height: 350px;
+    width: 175px;
+    text-align: start;
+    margin-right: 5px;
+    margin-top: 3px;
+    font-size: 18px;
+}
+div.container2{
+    display: flex;
+    flex: 20;
+    flex-direction: column;
+    color: white;
+    justify-content: flex-start;
+    font-size: 18px;
+}
+div.container3{
+    display: flex;
+    justify-content: flex-start;
+    height: 350px;
+    width: 280px;
+    background-color: rgb(124, 8, 8);
+}
+h3{
+    display: flex;
+    justify-content: center;
+    margin-top: 0;
+    background-color: rgb(189, 186, 186);
+}
+aside.title{
+    display: flex;
+    flex-direction: column;
+}
+div.line-top{
+    display: inline-block;
+    margin-left: 46px;
+    margin-right: 30px;
+}
+.line-top:before{
+    content: " ";
+    background: #8c8c8d;
+    width: 80px;
+    height: 1px;
+    position: absolute;
+}
+ul.left{
+    display: flex;
+    flex-direction: column;
+    line-height: 3;
+    margin-left: -30px;
+    color: orangered;
+    text-decoration: underline;
+    margin-top: 0px;
+    font-size: 18px;
+}
+footer{
+    background-color: orangered;
+    color: white;
+    text-align: center;
+    font-size: 20px;
+    margin-top: -60px;
+}
+span:before{
+    content: "\00A9 \00A0";
+}
```

Comment:
Also lets start to use something like minimal reset.css. Please add styles for body (here will be selector for body tag) to set its margin and padding to 0

Reply:
done)

Reply:
Don`t see any styles on body tag

Reply:
maybe u looked old file
in new index.html fixed ;)

Reply:
Just pulled changes. None styles on <body> tag. Only standard margin, that we need update to 0 :)
<img width="1728" alt="Screenshot 2024-09-20 at 23 12 56" src="https://github.com/user-attachments/assets/40a36ad7-4d33-4e59-acbc-ccbbf404404b">

Reply:
done)